Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9.0-alpine3.12
+RUN apk add --no-cache --update git=2.26.2-r0 gcc=9.3.0-r2 musl-dev=1.1.24-r10 libffi-dev=3.3-r2 g++=9.3.0-r2 \
```

Comment:
Update OpenSSL to make the bot work properly (Discord TLS issue)

Reply:
Wait crap that was meant to be suggest changes

Reply:
Can confirm this is required